Makefile

```diff
@@ -105,6 +109,21 @@ parsertrace_g: http_parser_g.o contrib/parsertrace.c
 tags: http_parser.c http_parser.h test.c
 	ctags $^
 
+install: library
+	install -D $(SONAME) $(LIBDIR)/$(SONAME)
+	ln -s $(LIBDIR)/$(SONAME) $(LIBDIR)/libhttp_parser.so
```

Comment:
Maybe place this after the install of the header so that erroring on an existing symlink doesn't leave a partially installed libhttp_parser (i.e. the library gets installed but not the headers.)